gatsby-node.js

```diff
@@ -152,19 +152,11 @@ exports.sourceNodes = async (
     };
 
     createNode(node);
-    return processedData.childNodes;
-  });
-
-  let flattenedChildNodes = await Promise.all(childNodes).then(nodes =>
-    nodes.reduce(
-      (accumulator, currentValue) => accumulator.concat(currentValue),
-      []
-    )
-  );
 
-  return Promise.all(flattenedChildNodes).then(nodes => {
-    nodes.forEach(node => createNode(node));
-  });
+    await Promise.all(processedData.childNodes).then(nodes => {
+      nodes.forEach(node => createNode(node));
+    });
+  }, { concurrency: 100 });
```

Comment:
```suggestion
  }, { concurrency: 5 });
```

Having run this a few times I actually did have the request block behavior with a concurrent request limit of 100 on one occasion, so knocking this down to 5 may be a meaningful change in spite of my initial doubts. I have not experienced any problems with a limit of 5.